src/build/templates/tasks/es5/manifest.js

```diff
@@ -18,6 +17,8 @@ gulp.task('manifest:prod', function(done) {
 })
 
 function build(done, extra) {
+  delete require.cache[require.resolve('../app/manifest.json')]
+  var manifests= require('../app/manifest.json')

```

Comment:
I prefer replacing `require()` altogether, with a call to [read-json-sync](https://github.com/shinnn/read-json-sync) or a similar module.